onnxruntime/contrib_ops/cuda/bert/ngram_repeat_block_impl.cu

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
The cuda implementation is from [microsoft/fastseq](https://github.com/microsoft/fastseq) : [ngram_repeat_block_cuda_kernel.cu](https://github.com/microsoft/fastseq/blob/main/fastseq/clib/cuda/ngram_repeat_block_cuda_kernel.cu)